### PR TITLE
Create MEMBER_TRAVEL_FUND.MD

### PR DESCRIPTION
**Pull request by hackygolucky**
As discussed in today's CommComm meeting, this creates the process about travel funds for the CommComm, since there is a separate scope of members than the TSC fund. 

A separate issue will be filed about the specific request for the 2017 travel fund allocation.

**ON HOLD**
This PR is on hold for further coordination with the @nodejs/tsc for a joint travel fund.

### CODE REVIEW

**Comment by RichardLitt** [comment]
LGTM.

File: MEMBER_TRAVEL_FUND.MD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give, if applicable.
+ * Include the size of the stipend you wish to receive.
+  * Reimbursement stipends are expected to vary with travel distance.
+* Once the final amount spent is known, update the table again with that information.
+
+The CommComm will consider each proposal in its next regular meeting. While it is
+ultimately at the CommComm's discretion who to approve the following considerations
+are made.
```
Comment (pup) [inline MEMBER_TRAVEL_FUND.MD]: shouldn't there be a colon here?

File: MEMBER_TRAVEL_FUND.MD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* Open a pull request against this repository which adds an entry to the table at the bottom of this file.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give, if applicable.
+ * Include the size of the stipend you wish to receive.
+  * Reimbursement stipends are expected to vary with travel distance.
+* Once the final amount spent is known, update the table again with that information.
+
+The CommComm will consider each proposal in its next regular meeting. While it is
+ultimately at the CommComm's discretion who to approve the following considerations
```
Comment (pup) [inline MEMBER_TRAVEL_FUND.MD]: i think there could be a comma after `approve`, for better readability

File: MEMBER_TRAVEL_FUND.MD [hunk 3]
```diff
@@ -0,0 +1,41 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation Community Committee(CommComm) to travel and spread knowledge about and support the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be CommComm Members of the Node.js project.
```
Comment (nebrius) [inline MEMBER_TRAVEL_FUND.MD]: I _think_ this should be rephrased to "Candidates must be a CommComm member, or a member of a team or Working Group overseen by CommComm", right?

**Comment by pup** [comment]
is this on the meeting agenda or can this be merged? /cc @nebrius

**Comment by nebrius** [comment]
@hackygolucky do we want to go ahead and merge this now, or wait until some of the meta-discussion around joint management is figured out first?

**Comment by hackygolucky** [comment]
This needs to be coordinated with @jasnell @mhdawson and the TSC. The result of the last Board meeting was that they'd prefer a jointly coordinated single travel fund. So we may end up needing to scrap this PR and link to a new joint one, wherever it ends up contained.

**Comment by pup** [comment]
now that the tsc travel fund has been made available for general members, isn't this PR obsolete?

**Comment by bnb** [comment]
@hackygolucky Is there any update on the status of this PR?

**Comment by jasnell** [comment]
Fyi... The travel fund is now jointly managed by the tsc and commcomm.
There is already a tracking document in the tsc repo that can continue to
be used

**Comment by bnb** [comment]
@hackygolucky can we close this issue, given the Travel Fund now lives in the nodejs/admin repo?

**Comment by hackygolucky** [comment]
We can def close this. See the [nodejs/admin](https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.MD#purpose) repo for filing for Travel Fund!

**Pull request closed without merging**